docs/examples/overlaypanel/defaultExample.js

```diff
@@ -10,7 +10,7 @@ import {
   Flex,
   Layer,
   RadioButton,
-  Sheet,
+  OverlayPanel,
```

Comment:
nit: alphabetize these changed imports plz (someday I'll make a lint rule w/autofix for this)

Reply:
done